packages/smooth_app/lib/pages/preferences/user_preferences_widgets.dart

```diff
@@ -236,8 +236,11 @@ class UserPreferencesTitle extends StatelessWidget {
         child: Text(
           label,
           style: Theme.of(context).textTheme.displaySmall?.copyWith(
-                height: 2.5,
-              ),
+              fontSize: 24,
+              fontWeight: FontWeight.bold,
+              color: Theme.of(context).brightness == Brightness.light
+                  ? const Color(0xFF000000)
+                  : const Color(0xFFFFFFFF)),
```

Comment:
Could you make it Colors.black and Colors.white, thats way easier to understand :D

Reply:
okay

Reply:
Made the changes @M123-dev